siliconcompiler/tools/openroad/sc_apr.tcl

```diff
@@ -118,15 +120,15 @@ read_lef  $sc_techlef
 # Read Targetlibs
 foreach lib $sc_targetlibs {
 	read_liberty [dict get $sc_cfg library $lib nldm typical lib]
-	read_lef [dict get $sc_cfg library $lib lef]
+	read_lef [dict get $sc_cfg library $lib lef $sc_stackup]
 }
 
 # Read Macrolibs
 foreach lib $sc_macrolibs {
     if {[dict exists $sc_cfg library $lib nldm]} {
         read_liberty [dict get $sc_cfg library $lib nldm typical lib]
     }
-    read_lef [dict get $sc_cfg library $lib lef]
+    read_lef [dict get $sc_cfg library $lib lef stackup]
```

Comment:
Should this be `$sc_stackup`?

Reply:
Makes sense, I will fix in the next round.